libs/core/include/core/vector.hpp

```diff
@@ -1,55 +0,0 @@
-#pragma once
```

Comment:
@baykaner

Reply:
Is this unused?

Reply:
It's only used by k_means.hpp, which is itself unused. Khan asked me not to delete k_means, as it's in need of a redesign, but a lot of it can be reused, so I left it alone.